lib/html/pipeline/dradis_escape_html_filter.rb

```diff
@@ -13,13 +13,19 @@ class Pipeline
     #
     # This filter does not write any additional information to the context hash.
     class DradisEscapeHTMLFilter < TextFilter
+      PROTOCOLS = %w{
+        ed2k ftp http https irc mailto news gopher nntp telnet webcal xmpp
+        callto feed svn urn aim rsync tag ssh sftp rtsp afs file
+      }
+
       def call
         text = ERB::Util.html_escape(@text)
 
-        # Match the text between bc. and bc.. following the textile rules
+        # Match the text under bc./bc.. and links, following the textile rules
         regex = Regexp.union(
           /(?<=bc. )(.*?)(?=(\r\n|\n){2})/m,
```

Comment:
Now that we are here, I think this this should be: 
```
/(?<=bc\. )(.*?)(?=(\r\n|\n){2})/m
```
(using `\.` instead of just `.` after the `bc`)
Also on the next line for the `bc..`.

If not we are matching undesired cases:
http://rubular.com/r/WGCnnGTAr3